.github/workflows/fbgemmci.yml

```diff
@@ -37,6 +36,7 @@ jobs:
         echo GITHUB_WORKSPACE  = $GITHUB_WORKSPACE
         echo GITHUB_SHA        = $GITHUB_SHA
         echo GITHUB_REF        = $GITHUB_REF
+        c++ --verbose
```

Comment:
Just for my education (I'm new to github action). A reason for adding this line?

Reply:
Just wan to print out some information. We can remove this later.